templates/includes/sidebar.html

```diff
@@ -0,0 +1,18 @@
+<ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+    User: {{ user.username }}
+    </li>
+    <li>
+      <a href="{% url 'logout' %}">Logout</a>
+    </li>
+    {% else %}
+    <li>
+      <a href="{% url 'login' %}">Login</a>
+    </li>
+  {% endif %}
+  <br>
+  <hr>
+  <li><a href="{% url "blog:index" %}">Home page</a></li>
+  <li><a href="{% url "blog:posts-list" %}">Posts</a></li>
```

Comment:
The URL template tag should use single quotes consistently. Consider changing the double quotes to single quotes for consistency with the rest of the file.